src/latexdocvisitor.cpp

```diff
@@ -1283,6 +1292,10 @@ void LatexDocVisitor::visitPre(DocHtmlCell *c)
   if (c->isHeading())
   {
     m_t << "\\cellcolor{\\tableheadbgcolor}\\textbf{ ";
+  } 
+  else 
+  {
+    if (c->isFirst()) m_t << "\\hspace{0pt}";
```

Comment:
I see that it is for the hyphenation of the words (http://sigstp.blogspot.com/2010/09/latex-hyphenate-single-long-words-in.html).
I think this should not be hidden into a proposed pull request for a completely unrelated problem and should be a separate proposed pull request.